tests/basic-tests.js

```diff
@@ -147,9 +146,9 @@ describe("Basic Tests", () => {
       const pathPrefix = "44'/9000'/0'";
       const pathSuffixes = ["0/0", "0/1", "1/100"];
       const ui = await flowMultiPrompt(this.speculos, [
-        [{header:"Sign",body:"Transaction"}],
-        [{header:"Transfer",body:"0.000012345 AVAX to fuji12yp9cc0melq83a5nxnurf0nd6fk4t224unmnwx"}],
-        [{header:"Fee",body:"0.123444444 AVAX"}],
+        [{header:"Sign",body:"Transaction"},
+         {header:"Transfer",body:"0.000012345 AVAX to fuji12yp9cc0melq83a5nxnurf0nd6fk4t224unmnwx"},
+         {header:"Fee",body:"0.123444444 AVAX"}],
         [{header:"Finalize",body:"Transaction"}],
```

Comment:
This is a great improvement. Even better if the `Finalize` prompt could be slipped in to the first array of prompts so that it's exactly one confirmation, but this already rules.

Reply:
The finalize one can't as easily be, but also do you want the finalize one? I will still have the sign yes/no wt the end. I think with the batched prompts the "finalize" isn't adding much.

Reply:
I didn't understand we will still have the yes/no at the end, so yeah, let's ditch "finalize", good idea